src/ops/euclid.rs

```diff
@@ -46,6 +46,14 @@ pub trait Euclid: Sized + Div<Self, Output = Self> + Rem<Self, Output = Self> {
     /// assert_eq!(Euclid::rem_euclid(&-a, &-b), 1);
     /// ```
     fn rem_euclid(&self, v: &Self) -> Self;
+
+    /// Returns both the euclidian division quotien and remainer
+    ///
+    /// By default, it internaly calls both `Euclid::div_euclid` and `Euclid::rem_euclid`,
+    /// but it can be overwritten in order to implement some optimization.
```

Comment:
```suggestion
    /// By default, it internally calls both `Euclid::div_euclid` and `Euclid::rem_euclid`,
    /// but it can be overridden in order to implement some optimization.
```

Reply:
done